src/app/accounts/register.component.ts

```diff
@@ -41,6 +41,27 @@ export class RegisterComponent extends BaseRegisterComponent {
         this.showTerms = !platformUtilsService.isSelfHost();
     }
 
+    getPasswordScoreAlertDisplay() {
+        if (this.enforcedPolicyOptions == null) {
+            return '';
+        }
+
+        switch (this.enforcedPolicyOptions.minComplexity) {
+            case 4:
+                return this.i18nService.t('strong') + ' (4)';
+            case 3:
+                return this.i18nService.t('good') + ' (3)';
+            case 2:
+                return this.i18nService.t('weak') + ' (2)';
+            case 1:
+                return this.i18nService.t('weak') + ' (1)';
+            case 0:
+                return this.i18nService.t('weak') + ' (0)';
+            default:
+                return '';
+        }
```

Comment:
```suggestion
        let str:string;
        switch (this.enforcedPolicyOptions.minComplexity) {
            case 4:
                str = this.i18nService.t('strong');
            case 3:
                str = this.i18nService.t('good');
            default:
                str = this.i18nService.t('weak')';
        }
        return str + ' ('+this.enforcedPolicyOptions.minComplexity+')';
```

could simplify this a bit